ntc_templates/templates/ipinfusion_ocnos_show_lldp_table.textfsm

```diff
@@ -0,0 +1,17 @@
+Value LOCAL_PORT (\S+)
+Value REMOTE_DEVICE (\S+)
```

Comment:
```suggestion
Value NEIGHBOR (\S+)
```

Reply:
fixed in [ef74743](https://github.com/networktocode/ntc-templates/pull/923/commits/ef74743e064a2984fd28a620dddc342f8ab1534f)